CHANGELOG.md

```diff
@@ -1,3 +1,147 @@
+<a name="2.0.0-beta.11"></a>
+# [2.0.0-beta.11 carapace-parapet](https://github.com/angular/material2/compare/2.0.0-beta.10...2.0.0-beta.11) (2017-09-21)
+
+
+### Highlights
+* Each `@angular/material` component is now bundled into its own javascript file. This will allow
+tools like webpack to more easily load _only_ the components being used in an application.
+* New stepper component! The base behavior lives in `@angular/cdk` with Material Design flavors in
+`@angular/material`.
+
+
+### Breaking changes
+
+* Angular Material now requires **Angular 4.4.3 or greater**
+* MaterialModule has been removed. (#6803) ([cf1ece0](https://github.com/angular/material2/commit/cf1ece0))
+[See the deprecation notice from beta.3 for more information](https://github.com/angular/material2/blob/master/CHANGELOG.md#200-beta3-cesium-cephalopod-2017-04-07).
+* `MdCoreModule` has been removed. Most of its functionality has been move to `@angular/cdk` over
+the last few released.
+* `FocusOriginMonitor` has been renamed to `FocusMonitor` and moved to `@angular/cdk`.
+* **chip-list:**  - The outputs `select` and `deselect` have been removed in favor of a single
+  `onSelectionChange` output.
+* **viewport-ruler:** Previously the `ScrollDispatcher.scrolled` subscription would react both on
```

Comment:
I don't think this got merged, there were some issues with that PR. It might have been merged and then reverted